tests/interfaces/test_qnode_torch.py

```diff
@@ -204,10 +214,16 @@ def circuit(a, b):
         assert np.allclose(a.grad, expected[0], atol=tol, rtol=0)
         assert np.allclose(b.grad, expected[1], atol=tol, rtol=0)
 
-        spy.assert_called()
+        if diff_method == "finite-diff":
+            spy.assert_called()
+        elif diff_method == "backprop":
+            spy.assert_not_called()
 
     def test_jacobian_dtype(self, dev_name, diff_method, tol):
         """Test calculating the jacobian with a different datatype"""
+        if diff_method == "backprop":
+            pytest.skip("Test does not support backprop")

```

Comment:
Also curious: how come this doesn't work in backprop?

Reply:
Same reason as above; setting the dtype is a feature of the interface, however backprop does not use the interface